bdk_chain/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bdk_chain"
-version = "0.1.1"
+version = "0.2.0"
```

Comment:
In general you shouldn't change the package version in this kind of PR.

Are you just doing this because you're making releases at the same time? I see that `bdk_chain` on crates.io is already at `0.3.0`.

Reply:
I had to release it for use with the other crates within the repo. Also, I was using the crate for the BDK PR as well (So I needed a new release with the persistence stuff).